### ISSUE TITLE
Non-recursive includePath does not work

### ISSUE THREAD

**Issue by janjongboom** [microsoft/vscode-cpptools#546]
According to #475 you can define includePaths to be non-recursive by adding `/*` at the end of the folder name. This does not work for me on VsCode 1.10.2 with C/C++ plugin 0.10.3.

My vscode configuration is here: https://gist.github.com/janjongboom/13e96a66b5b167597c70070b15c71696

And when I type `D5` in my editor it shows a symbol from a file which is not in this list...

![screen shot 2017-03-10 at 09 37 25](https://cloud.githubusercontent.com/assets/816856/23788183/b1acdef0-0575-11e7-95f0-24b072d6d267.png)

/cc @bobbrow @sg-

**Comment by bobbrow** [comment]
Did you modify the includePath for a folder that was already open in VS Code?  We don't currently remove symbols for directories after they've already been parsed - even if you change the includePath.  You might need to delete the DB and have the extension rescan your folder using the new includePath.  Sorry for the annoyance.  We're working on making this better.

Based on your gist, it looks like you are developing on Mac.  So you should be able to find the DB at this location:
```
~/Library/Application Support/Code/User/workspaceStorage/{hash}/ms-vscode.cpptools/.browse.VC.db
```

**Comment by sean-mcmanus** [comment]
Unfortunately, the "*" is not working on Linux/Mac -- it works on Windows. The bug should be easy to fix for the next release.

**Comment by janjongboom** [comment]
@sean-mcmanus @bobbrow I have the same issue on Windows... Also get symbols that are not in my includePaths even though all my paths are postfixed with `/*`. I have removed my DB... Any clue?

**Comment by sean-mcmanus** [comment]
That's strange. I'm not able to repro the issue on Windows. There are additional scenarios where files/symbols will get added to the database, such as when a file is opened or when files are created in the workspace. Also, when you remove paths from the includePath, we don't remove the files/symbols from the database, which we plan to fix soon. Can you give an example of an include path and file/symbol that is reproducing the error, such as from the MSVC or Windows SDKs?

**Comment by janjongboom** [comment]
@sean-mcmanus I was on an old version of C/C++ extension on my Windows installation that didn't have the `addWorkspaceRootToIncludePath` setting. Now fixed (on Windows, not on macOS).

**Comment by bobbrow** [comment]
This issue should have been fixed in 0.11.1.  Please let us know if you are still seeing this on the latest version of the extension.  Thanks!
